### ISSUE TITLE
The Accessibility Epic

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#182]
FreeBMD2 should meet the  Web Content Accessibility Guidelines (https://www.w3.org/WAI/standards-guidelines/wcag/).

WACG defines conformance at three levels, A, AA, and AAA.  FreeBMD2 should be at least AA accessible.

In addition, FreeBMD2 should be meet the needs of users, even where they are not covered by the WCAG standards (e.g. "the white space issue").

The report conducted on FreeREG in 2016, which determined that it met, at that time, only 9 of the 16 AA standards, provides information that will be of use in implementing the multi-modal code on FreeBMD2 https://drive.google.com/open?id=0BzRDUeF70MtpZ196MDNUMG5peU0
